source/_integrations/weather.template.markdown

```diff
@@ -34,11 +34,38 @@ weather:
     temperature_template: "{{ states('sensor.temperature') | float }}"
     temperature_unit: "°C"
     humidity_template: "{{ states('sensor.humidity') | float }}"
-    forecast_daily_template: "{{ state_attr('weather.my_region', 'forecast') }}"
+    forecast_daily_template: "{{ state_attr('sensor.daily_weather_forecast', 'forecast') }}"
 ```
 
 {% endraw %}
 
+If you want to use the forecast of another weather integration, you'll need to create a trigger based template sensor and store the forecast in an attribute.
+This is an example on how you can create such a sensor which will update every 3 hours.
+
+{% raw %}
+
+```yaml
+# Example configuration.yaml entry
+template:
+  - trigger:
+      - platform: time_pattern
+        hours: "/3"
+    action:
+      - service: weather.get_forecasts
+        target:
+          entity_id: weather.my_region
+        data:
+          type: daily
+        response_variable: forecast
+    sensor:
+       - unique_id: daily_weather_forecast_template_sensor
+         name: Daily Weather Forecast
+         state: "{{ now() }}"
+         attributes:
+           forecast: "{{ forecast['weather.my_region'].forecast }}"
+```
+{% endraw %}
```

Comment:
This is basically re-creating what we wanted to get rid off. I don't think it makes sense to do that?

Reply:
Yes, you are right. 

My main goal was to remove the reference to `state_attr('weather.my_region', 'forecast')` as that was no longer possible.
But if you do want to add a forecast to a template weather, it has to come from somewhere, that's why I added it. 

I can remove that part, and leave it to the end user how they get the data for the `<type>_forecast_template`s.